src/libraries/System.Text.RegularExpressions/gen/System.Text.RegularExpressions.Generator.csproj

```diff
@@ -17,8 +17,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <!-- This generator ships as part of the .NET ref pack. Since VS ingests new Roslyn versions at a different cadence than the .NET SDK, we are pinning this generator to build against Roslyn 4.0 to ensure that we don't break users who are using .NET 7 previews in VS. -->
```

Comment:
Given that we had this explicit remark section here, are we changing plans now? I understand the morivtion behind this change but will this effectively force our nighlty build consumers and/or our repo contributors to use an int Preview build of VS (which they likely not have access to)?

cc @stephentoub for Regex

Reply:
No, the LatestVS version tracks the most recent public preview that we need to use APIs from.

I removed the remark as just using the LatestVS version implies the exact same plan (it's the whole reason we introduced it in our infrastructure).